### PR TITLE
mkcloud_reserve_pool: use arithmetic with seconds

### PR DESCRIPTION
**Pull request by bmwiedemann**
allows to specify notification delays longer than a day
and crossing midnight

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
+1 (if tested)

File: scripts/mkcloudhost/mkcloud_reserve_pool [hunk 1]
```diff
@@ -2,67 +2,49 @@
...
+delay=$3
 message="Did you free VM ${pool_vm} on mkch${host}.cloud.suse.de ?"
 
 if ! [[ $(which xmessage) ]]; then
   echo "please install xmessage"
   exit 1
 fi
 
-if [[ -z $1 ]] || [[ -z $2 ]] || [[ -z $3 ]]
+if [[ -z $1 ]] || [[ -z $2 ]]
```
Comment (jdsn) [inline scripts/mkcloudhost/mkcloud_reserve_pool]: `if [[ ! $1 ||  ! $2 ]]`

File: scripts/mkcloudhost/mkcloud_reserve_pool [hunk 2]
```diff
@@ -2,67 +2,49 @@
...
-case "$4" in
-     [0-9] | [0-5][0-9] ) minutes="$4"
-                          shift 2 ;;
-     * ) shift 1
-         minutes="" ;;
-esac
-
-if test -z "$minutes"
-then echo "Enter minutes (random input or timeout after 10 seconds set minutes to 0): "
+if test -z "$delay"
```
Comment (jdsn) [inline scripts/mkcloudhost/mkcloud_reserve_pool]: `if [[ $delay ]]`

File: scripts/mkcloudhost/mkcloud_reserve_pool [hunk 3]
```diff
@@ -2,68 +2,50 @@
...
-
-if test -z "$minutes"
-then echo "Enter minutes (random input or timeout after 10 seconds set minutes to 0): "
-     read -t 10 minutes
-     case "$minutes" in
-          [0-9] | [0-5][0-9] ) ;;
-          * ) minutes="0" ;;
+if [[ ! $delay ]]
+then echo "Enter reminder delay (random input or timeout after 10 seconds set delay to 60m): "
+     read -t 10 delay
```
Comment (jdsn) [inline scripts/mkcloudhost/mkcloud_reserve_pool]: you can use `-p` for the prompt string

File: scripts/mkcloudhost/mkcloud_reserve_pool [hunk 4]
```diff
@@ -2,68 +2,50 @@
...
+     case "$delay" in
+          [0-9]* ) ;;
+          * ) delay="60m" ;;
```
Comment (jdsn) [inline scripts/mkcloudhost/mkcloud_reserve_pool]: shorter:
`: ${delay:=60m}`

File: scripts/mkcloudhost/mkcloud_reserve_pool [hunk 5]
```diff
@@ -2,68 +2,47 @@
...
+delay=$3
 message="Did you free VM ${pool_vm} on mkch${host}.cloud.suse.de ?"
 
 if ! [[ $(which xmessage) ]]; then
   echo "please install xmessage"
   exit 1
 fi
 
-if [[ -z $1 ]] || [[ -z $2 ]] || [[ -z $3 ]]
+if [[ ! $1 || ! $2 ]]
```
Comment (jdsn) [inline scripts/mkcloudhost/mkcloud_reserve_pool]: looks nicer:
`if [[ ! $host || ! $pool_vm ]]`

**Comment by jdsn** [comment]
:+1:

**Pull request merged**